README.md

```diff
@@ -1 +1,68 @@
-# microverse-capstone-one
\ No newline at end of file
+# Conference Website Capstone
+
+This project contains the HTML, CSS and JavaScript Capstone Project. Capstone Projects are done at the end of a module at Microverse.
+
+
+![screenshot](./images/conf-desk.PNG)
+
+![screenshot](./images/conf-mob.PNG)
+
+ 
+
+## Live Demo
+
+[Live Demo Link](https://codecaiine.github.io/conference-website/)
+
+[Demo video link](https://www.loom.com/share/0031406c6c2b4a87b2066580a29365fb)
```

Comment:
- [ ] Nice job by adding video description of your project as requested by the previous code reviewer. However, your video description **not detailed**. Kindly ensure that video description also covers some salient aspect your **code base** for this project. (Video should be at least 4mins and max of 5 mins)